.buildkite/publish_tutorials_output.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Ensure that our git wants to talk to github without prompting
+ssh-keyscan github.com >> ~/.ssh/known_hosts
+
+# Clone TuringTutorials to temporary directory
+temp_dir=$(mktemp -d)
+git -C "${temp_dir}" clone --branch artifacts git@github.com:TuringLang/TuringTutorials .
```

Comment:
I guess a possibly cleaner and more public (and hence easier discoverable?) alternative would be to push to a separate repo such as TuringTutorialsOutput (SciMLTutorials pushes to SciMLTutorialsOutput).